apps/teamsjs-diagnostic-app/package.json

```diff
@@ -12,6 +12,7 @@
     "@microsoft/teamsfx-react": "^3.0.0",
     "@testing-library/dom": "^10.0.0",
     "axios": "^0.21.1",
+    "jspdf": "^2.5.1",
```

Comment:
Since you're no longer using pdf files, can this dependency be removed?

Reply:
Fixed!